core-concepts/displaying-data-read/README.md

```diff
@@ -38,6 +38,13 @@ A widget must have a unique name that acts as an identifier on the page. It is u
 {{ Table1.selectedRow.id }}
 ```
 
+### Grouping Widgets
+
+Widgets can be grouped together by selecting multiple widgets with Ctrl + Left Click. Once selected, click on the dotted square icon or press Ctrl + G. 
+Once grouped, the widgets are put in a container and can move together.
+
+![](../../.gitbook/assets/group-widgets.gif)
```

Comment:
Would you please check this gif once? I don't see it loading on the page.

Reply:
Strange. The GIF is rendering when viewed in my forked repo here: https://github.com/secure-crypto/appsmith-docs/tree/docx-add-group-widget/core-concepts/displaying-data-read. But not showing when making a PR. I tried with a smaller file size GIF but it's the same. Any thoughts on what I can try next @Pranay105 ?